packages/api/methods.ts

```diff
@@ -192,3 +192,27 @@ export function updateRule(rule) {
 export function deleteRule(id) {
   return send('api/rule-delete', { id });
 }
+
+export function getSchedules() {
+  return send('api/schedules-get');
+}
+
+export function createSchedule(schedule, conditions) {
+  return send('api/schedule-create', { schedule, conditions });
+}
+
+export function updateSchedule(schedule, conditions, resetNextDate) {
+  return send('api/schedule-update', { schedule, conditions, resetNextDate });
+}
+
+export function deleteSchedule(id) {
+  return send('api/schedule-delete', { id });
+}
+
+export function scheduleSkipNextDate(id) {
+  return send('api/schedule-skip-next-date', { id });
+}
+
+export function scheduleGetUpcomingDates(config, count) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what's `config`? How is it different from `count`? Could `count` be part of `config`? What if I don't want to provide it, but only want to provide `count`? 

I know you made it to match the internal API right now. But the internal API is easy to change (later on). Whereas the public API is immutable unless we create a breaking change release (which we ideally shouldn't do). So the external API interface is king and we can worry about the internals later on.